### ISSUE TITLE
unrecognized selector sent to instance when using `NetworkLogger.setNetworkDataObfuscationHandler` 

### ISSUE THREAD

**Issue by saleksandras** [Instabug/Instabug-React-Native#630]
```
Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[__NSDictionaryM dataUsingEncoding:]: unrecognized selector sent to instance
```

### Steps to Reproduce the Problem

Add following code to one as to obfuscate response body as suggested by the [documentation](https://docs.instabug.com/docs/react-native-logging#obfuscating-data)

```
NetworkLogger.setNetworkDataObfuscationHandler(async networkData => {
  try {
    networkData.responseBody = { ...JSON.parse(networkData.responseBody), token: '********' };
  } catch (e) {}
  return networkData;
});
```

Removing `NetworkLogger.setNetworkDataObfuscationHandler(async networkData => {});`  block solves the issue.

### Expected Behavior

App works without crashing and logs being modified 

### Actual Behavior

Getting `Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[__NSDictionaryM dataUsingEncoding:]: unrecognized selector sent to instance` error message

### SDK Version

"instabug-reactnative": "10.8.1"

### React Native, iOS and Android Versions

"react-native": "0.65.1"
Xcode 12.5.1 and 13.0.0
iOS 14.5

**Comment by saleksandras** [comment]
instabug-reactnative 10.4.0 doesn't have this issue

**Comment by AliAbdelfattah** [comment]
Hey @saleksandras, we're releasing a fix that should handle this issue. Would it be possible to verify it fixes your issue please?

```
npm install --save https://github.com/Instabug/Instabug-React-Native#snapshot-ios/10.9.2
```

**Comment by saleksandras** [comment]
@AliAbdelfattah Thanks! It solves this issue

**Comment by AliAbdelfattah** [comment]
@saleksandras thank you for the feedback. The fix was included in the recent release 10.9.0.
